edge/pkg/eventbus/mqtt/client.go

```diff
@@ -58,9 +58,11 @@ var (
 
 // Client struct
 type Client struct {
-	MQTTUrl string
-	PubCli  MQTT.Client
-	SubCli  MQTT.Client
+	MQTTUrl  string
+	Username string
+	Password string
+	PubCli   MQTT.Client
+	SubCli   MQTT.Client
 }
```

Comment:
Do you consider supporting custom configuration of `clientID`? 
There may be a demand?

Reply:
clientID is generated by code like this, do we need to make it configurable?
https://github.com/kubeedge/kubeedge/blob/ea4dc19408cf6b424632b93a33c1aa52405b6090/edge/pkg/eventbus/mqtt/client.go#L143-L144

Reply:
> 
done
add clientIDs for subClient and pubClient